### ISSUE TITLE
v2.2.0 logging is missing "CycloneDX: Parameters"

### ISSUE THREAD

**Issue by msymons** [CycloneDX/cyclonedx-maven-plugin#83]
It seems that logging for `makeAggregateBom` is not outputting the complete content.  For v2.2.0 (showing that new functionality to output the path and filename is working):
```
 --- cyclonedx-maven-plugin:2.2.0:makeAggregateBom (default-cli) @ xxx ---
[INFO] CycloneDX: Creating BOM
[INFO] CycloneDX: Writing BOM (XML): /data/jenkins/workspace/xxx/target/bom.xml
[INFO] CycloneDX: Validating BOM (XML): /data/jenkins/workspace/xxx/target/bom.xml
[INFO] CycloneDX: Writing BOM (JSON): /data/jenkins/workspace/xxx/target/bom.json
[INFO] CycloneDX: Validating BOM (JSON): /data/jenkins/workspace/xxx/target/bom.json
```
Compare with output from `makeBom` from comment on issue #80 
```
[INFO] --- cyclonedx-maven-plugin:2.2.0-SNAPSHOT:makeBom (default-cli) @ project-1 ---
[INFO] CycloneDX: Parameters
[INFO] ------------------------------------------------------------------------
[INFO] schemaVersion          : 1.2
[INFO] includeBomSerialNumber : true
[INFO] includeCompileScope    : true
[INFO] includeProvidedScope   : true
[INFO] includeRuntimeScope    : true
[INFO] includeTestScope       : false
[INFO] includeSystemScope     : true
[INFO] includeLicenseText     : false
[INFO] outputFormat           : all
[INFO] outputName             : bom
[INFO] ------------------------------------------------------------------------
[INFO] CycloneDX: Resolving Dependencies
[INFO] CycloneDX: Creating BOM
[INFO] CycloneDX: Writing BOM (XML): /Users/steve/Development/CycloneDX/cyclonedx-maven-plugin/src/test/resources/project1/target/bom.xml
[INFO] CycloneDX: Validating BOM (XML): /Users/steve/Development/CycloneDX/cyclonedx-maven-plugin/src/test/resources/project1/target/bom.xml
[INFO] CycloneDX: Writing BOM (JSON): /Users/steve/Development/CycloneDX/cyclonedx-maven-plugin/src/test/resources/project1/target/bom.json
[INFO] CycloneDX: Validating BOM (JSON): /Users/steve/Development/CycloneDX/cyclonedx-maven-plugin/src/test/resources/project1/target/bom.json
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
```

**Comment by msymons** [comment]
I have tested that the defect is still present in v2.4.1  using goal `makeAggregateBom`, and that `makeBom` is fine:

```
[INFO] --- cyclonedx-maven-plugin:2.4.1:makeBom (cyclonedx-aggregate) @ xxx ---
[INFO] CycloneDX: Parameters
[INFO] ------------------------------------------------------------------------
[INFO] schemaVersion          : 1.2
[INFO] includeBomSerialNumber : true
[INFO] includeCompileScope    : true
[INFO] includeProvidedScope   : true
[INFO] includeRuntimeScope    : true
[INFO] includeTestScope       : true
[INFO] includeSystemScope     : true
[INFO] includeLicenseText     : false
[INFO] outputFormat           : xml
[INFO] outputName             : bom
[INFO] ------------------------------------------------------------------------
[INFO] CycloneDX: Resolving Dependencies
[INFO] CycloneDX: Creating BOM
[INFO] CycloneDX: Writing BOM (XML): /home/jenkins/workspace/xxx-BOM/target/bom.xml
[INFO] CycloneDX: Validating BOM (XML): /home/jenkins/workspace/xxx-BOM/target/bom.xml
```

**Comment by stevespringett** [comment]
I do not know how to fix. Maven is choosing not to log this all by itself.

**Comment by ThomGeG** [comment]
If we're talking about the missing logging for the parameters, it's just that the MOJO for `makeBom` is the only one calling the function that would log it.

I've already got a change to add the necessary call on a fork of mine ([here](https://github.com/ThomGeG/cyclonedx-maven-plugin/commit/c250ad13191cd319ec9b6869b58e9189aacb9631)), but can't raise a PR until I finish the other work I'm doing trying to solve #116

**Comment by msymons** [comment]
@stevespringett, I can confirm that this issue is now addressed and can be closed with a fix version of 2.7.0

```
 [INFO] --- cyclonedx-maven-plugin:2.7.0:makeAggregateBom (default-cli) @ ims ---
 [INFO] CycloneDX: Parameters
 [INFO] ------------------------------------------------------------------------
 [INFO] schemaVersion          : 1.4
 [INFO] includeBomSerialNumber : true
 [INFO] includeCompileScope    : true
 [INFO] includeProvidedScope   : true
 [INFO] includeRuntimeScope    : true
 [INFO] includeTestScope       : false
 [INFO] includeSystemScope     : true
 [INFO] includeLicenseText     : false
 [INFO] outputReactorProjects  : true
 [INFO] outputFormat           : all
 [INFO] outputName             : bom
 [INFO] ------------------------------------------------------------------------
```
